events/kafka_test.go

```diff
@@ -20,21 +20,25 @@ func TestKafkaEventMarshaling(t *testing.T) {
 		t.Errorf("could not unmarshal event. details: %v", err)
 	}
 
+	assert.Equal(t, inputEvent.BootstrapServers, "b-2.demo-cluster-1.a1bcde.c1.kafka.us-east-1.amazonaws.com:9092,b-1.demo-cluster-1.a1bcde.c1.kafka.us-east-1.amazonaws.com:9092")
+	assert.Equal(t, inputEvent.EventSource, "aws:kafka")
+	assert.Equal(t, inputEvent.EventSourceARN, "arn:aws:kafka:us-west-2:012345678901:cluster/ExampleMSKCluster/e9f754c6-d29a-4430-a7db-958a19fd2c54-4")
 	for _, records := range inputEvent.Records {
 		for _, record := range records {
 			utc := record.Timestamp.UTC()
 			assert.Equal(t, 2020, utc.Year())
+			assert.Equal(t, record.Key, "OGQ1NTk2YjQtMTgxMy00MjM4LWIyNGItNmRhZDhlM2QxYzBj")
+			assert.Equal(t, record.Value, "OGQ1NTk2YjQtMTgxMy00MjM4LWIyNGItNmRhZDhlM2QxYzBj")
+			
+			for _, header := range record.Headers {
+				for key, value := range header {
+					assert.Equal(t, key, "headerKey")
+					var headerValue string = string(value)
+					assert.Equal(t, headerValue, "headerValue")
+				}
+			}
 		}
 	}
-
-	// 3. serialize to JSON
-	outputJson, err := json.Marshal(inputEvent)
-	if err != nil {
-		t.Errorf("could not marshal event. details: %v", err)
-	}
-
-	// 4. check result
-	assert.JSONEq(t, string(inputJson), string(outputJson))
```

Comment:
The Kafka header is a list of map between string and []byte. 
According to [Marshal JSON](https://pkg.go.dev/encoding/json#Marshal)
```
Array and slice values encode as JSON arrays, except that []byte encodes as a base64-encoded string, and a nil slice encodes as the null JSON value.
```
This assertion won't be true anymore.